tests/test_utils.py

```diff
@@ -417,6 +419,61 @@ def test_convert_dict_to_env_variables(self):
             valid_env_items = convert_dict_to_env_variables(env)
         assert valid_env_items == ["ACCELERATE_DEBUG_MODE=1\n", "OTHER_ENV=2\n"]
 
+    def test_deprecated(self):
```

Comment:
I'm wondering if the decorator would also work for a class? If yes, let's add a test and update the docstring. If not, we can leave it as is.

Reply:
Done in [89b1974](https://github.com/huggingface/accelerate/pull/3161/commits/89b1974c0539bbcf21d70abf91505021d8991de4). Thank  you!